site/docs/encoding.md

```diff
@@ -41,7 +43,8 @@ Mark properties channels map data fields directly to visual properties of the ma
 | Property      | Type          | Description    |
 | :------------ |:-------------:| :------------- |
 | x, y          | [ChannelDef](#def)| X and Y coordinates for `point`, `circle`, `square`, `line`, `text`, and `tick`. (or to width and height for `bar` and `area` marks). |
-| color         | [ChannelDef](#def)| Color of the marks – either fill or stroke color based on mark type. (By default, fill color for `area`, `bar`, `tick`, `text`, `circle`, and `square` /   stroke color for `line` and `point`.) (See [scale range](scale.html#range) for more detail about color palettes.)  |
+| column, row  | [ChannelDef](#def)| `row` and `column` are special encoding channels for [faceting](#facet). |
```

Comment:
For here, since they are **special encoding channels**, let's put them at the end.  
(I think it's okay to make this slightly different from the spec above.)